cnf-tests/testsuites/e2esuite/gatekeeper/gatekeeper.go

```diff
@@ -52,7 +52,7 @@ var _ = Describe("gatekeeper", func() {
 
 		for _, namespace := range namespacesUsed {
 			if namespaces.Exists(namespace, client) {
-				err := client.Namespaces().Delete(context.Background(), namespace, metav1.DeleteOptions{})
+				err := namespaces.Create(namespace, client.CoreV1Interface)
```

Comment:
Why create instead of delete if the namespace already exists?
In line 71, we wait until the namespaces are deleted so with this change that check would fail, no?